### PR TITLE
post: portus 2.3 release

### PR DESCRIPTION
**Pull request by mssola**
Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

**Comment by mssola** [comment]
Do not merge this yet, since the final release has not been tagged 😉

File: _posts/2018-01-22-portus-2.3.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+It's been almost a year since we released Portus 2.2, but we can finally present
+to you the latest release of Portus. We wanted to deliver this new release much
+sooner, but some late-coming features changed the release schedule quite a bit,
+so we had to spend more time for the final release.
+
+For this release we have improved lots of small UX details and we have fixed as
+many bugs as we could from the 2.2 release. You can check the
+[changelog](https://github.com/SUSE/Portus/blob/master/CHANGELOG.md) for more
+info of this release, but in this blog post we want to highlight the following
```
Comment (vitoravelino) [inline _posts/2018-01-22-portus-2.3.md]: `... for more details, but...`

File: _posts/2018-01-22-portus-2.3.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+new features:
```
Comment (vitoravelino) [inline _posts/2018-01-22-portus-2.3.md]: Just `features`?

File: _posts/2018-01-22-portus-2.3.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+#### Security scanning
+
+Portus is now able to scan security vulnerabilities on your Docker images. This
+is done with different backends, where the stable one is [CoreOS
```
Comment (vitoravelino) [inline _posts/2018-01-22-portus-2.3.md]: `The scanner we tested and recommend is CoreOS Clair but you can also use a different backend for it. You have to enable...`

Rephase to something like that? I dunno...

File: _posts/2018-01-22-portus-2.3.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+`master` branch is not supported).
+
+You can read the [blog
+post](http://port.us.org/2017/07/19/security-scanning.html) or the
+[documentation](/features/6_security_scanning.html) for more info.
+
+#### Background process
+
+One of the main issues for Portus was that sometimes it took too long to
+complete certain critical tasks. For this release we have moved these tasks
```
Comment (vitoravelino) [inline _posts/2018-01-22-portus-2.3.md]: `... tasks. We have moved these...`

Comment (vitoravelino) [reply _posts/2018-01-22-portus-2.3.md]: `those`*

File: _posts/2018-01-22-portus-2.3.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+into a separate *background* process. This background process resides in the
+`bin/background.rb` file, and it can be enabled for containerized deployments
+by setting the `PORTUS_BACKGROUND` environment variable to true.
```
Comment (vitoravelino) [inline _posts/2018-01-22-portus-2.3.md]: Wrap true with "`".

File: _posts/2018-01-22-portus-2.3.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+
+The following tasks have been moved into this new process:
+
+- *Security scanning*: after testing security scanning more in depth, we noticed
+  that sometimes it could block Portus when showing the main page for
```
Comment (vitoravelino) [inline _posts/2018-01-22-portus-2.3.md]: It was blocking everything, not only the repositories page.

File: _posts/2018-01-22-portus-2.3.md (hunk 3 above)
Comment (mssola) [reply _posts/2018-01-22-portus-2.3.md]: I don't know, the other ones are not usable as of now 😅

Comment (vitoravelino) [reply _posts/2018-01-22-portus-2.3.md]: Then we should not mention `different backends`. It's not sounding good to me but I dunno how to help with that. :(

**Comment by boyand** [comment]
hey @mssola. What is the current plan about finalising the 2.3 release ?

**Comment by mssola** [comment]
> hey @mssola. What is the current plan about finalising the 2.3 release ?

We are updating documentation and investigating issues reported by users testing the `2.3` branch (and hence the `2.3` docker tag). We should be releasing rather soonish a final 2.3 release.

**Comment by mssola** [comment]
Closing in favor of #1707.

**Pull request closed without merging**